### PR TITLE
Fix part of #2399: Reformatting dimens.xml structure

### LINKED ISSUE #2399: Reformatting required in dimension file for better structure
Currently the dimensions in the **dimens.xml files** are segregated in groups (all having same values). The file is too large so we must rearrange the dimensions to form the groups on the base of their name, so that we find all the the dimensions related to a particular category at once. 

**Reference**
Category structure - https://github.com/oppia/oppia-android/pull/2111#discussion_r523663618

### PR DESCRIPTION
**Pull request by kartikeysaran**
## Explanation
Fixes part of #2399 
Dimens.xml was properly structured in the file

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Arjupta** [comment]
@kartikeysaran the tests are failing

**Comment by kartikeysaran** [comment]
> @kartikeysaran the tests are failing

I will review the code

**Comment by kartikeysaran** [comment]
@rt4914 All the test have been passed if reviewed this PR can be merged

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -214,157 +50,350 @@
...
+  <dimen name="font_scale_content_extra_large_text_view_size">28dp</dimen>
+  <dimen name="font_scale_content_text_size">20dp</dimen>
+  <dimen name="font_scale_content_small_text_view_size">16dp</dimen>
+  <dimen name="font_scale_content_large_text_view_size">24dp</dimen>
+
+  <!-- Fade -->
+  <integer name="fade_in_duration_ms">1000</integer>
+  <integer name="fade_out_duration_ms">1000</integer>
+
+  <!--State-->
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml:328]: Nit update everywhere:
There should be space before and after words.
`<!-- State -->`

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -214,157 +50,350 @@
...
+  <dimen name="profile_view_already_added_margin">20dp</dimen>
+  <dimen name="profile_view_not_added_margin">20dp</dimen>
+
+  <!--Walkthrough-->
+  <dimen name="walkthrough_final_no_button_margin_end">8dp</dimen>
+  <dimen name="walkthrough_topic_summary_container_margin">20dp</dimen>
+  <dimen name="walkthrough_final_yes_button_margin_start">8dp</dimen>
+  <dimen name="walkthrough_final_fragment_card_content_padding">24dp</dimen>
+
+  <!-- -->
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml:367]: Remove this.

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -214,157 +50,350 @@
...
+  <dimen name="feedback_item_question_view_margin_top">24dp</dimen>
+  <dimen name="next_button_question_view_padding_end">24dp</dimen>
 
-  <dimen name="story_chapter_card_playable_state_elevation">4dp</dimen>
-  <dimen name="ongoing_topic_card_elevation">4dp</dimen>
-  <dimen name="story_chapter_card_not_playable_state_elevation">2dp</dimen>
-  <dimen name="profile_chooser_padding_bottom_profile_already_added">224dp</dimen>
-
+  <!--Exploration-->
+  <dimen name="previous_button_item_exploration_split_view_margin_start">48dp</dimen>
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml:157]: Break these into multiple parts like 
`Previous Button`
`Previous Responses`
`Content Item`
`Submit Button`
`Submitted Answer`
`Return to Topic`
`Replay Button`
`Feedback Item`
`Continue Interaction Item`
`Continue Navigation Item`

etc. The main criteria is if there is an `xml` file then it should have its separate section like above names.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Update Title and Description to `Fix part of #bug_number....` considering there are other dimen's files too which will need this re-formatting.

**Comment by kartikeysaran** [comment]
@rt4914  I will update the code with the changes .

File: app/src/main/res/values/dimens.xml (hunk 3 above)
Comment (kartikeysaran) [reply app/src/main/res/values/dimens.xml:157]: Do i have to subdivide inside every format like  <!-- Exploration > <!--Previous Button><dimen name="previous"...../> <!--Previous response>......

Comment (rt4914) [reply app/src/main/res/values/dimens.xml:157]: Just update your branch with latest develop and there you will see the correct format at the bottom that I have followed for `content-item`

**Comment by Arjupta** [comment]
@kartikeysaran any update on the suggested changes?

**Comment by kartikeysaran** [comment]
@rt4914 I am busy with my college work this week if you want you can assign this issue to someone else . Thankyou !

**Comment by rt4914** [comment]
No issues. Thanks @kartikeysaran . Feel free to reach out again.

**Pull request closed without merging**